docs/DRONECAN.md

```diff
@@ -18,37 +18,49 @@ In order to enable also the serial (MAVLink) communication via DroneCAN, one nee
 
 The following parameters need to be set to enable MAVLink and RC over DroneCAN.
 
-Setup the CAN driver, protocol and enable the virtual serial port, then reboot the flight controller:
+Setup the CAN driver, protocol and enable the virtual serial support, then reboot the flight controller:
 
 - CAN_P1_DRIVER = 1
-- CAN_D1_PROTOCOL = 1 (set to 1 by default)
+- CAN_D1_PROTOCOL = 1 (is set to 1 by default)
 - CAN_D1_UC_SER_EN = 1
 
-Configure the DroneCAN virtual serial port:
+Configure the DroneCAN virtual serial port (mandatory settings):
 
 - CAN_D1_UC_S1_IDX = 0
 - CAN_D1_UC_S1_NOD = 68
 - CAN_D1_UC_S1_PRO = 2
+- CAN_D1_UC_S1_BD = 57 (57600)
 
-Adjust the DroneCAN streams:
+Adjust the DroneCAN streams (optional settings):
 
 - CAN_D1_UC_NTF_RT = 1
 - CAN_D1_UC_SRV_RT = 0
 
-Note: This assumes that you are not using DroneCAN for servo outputs.
+***Note***: This assumes that you are not using DroneCAN for servo outputs.
 
 Enable RC Channels via DroneCAN:
 
 - RC_PROTOCOLS bitmask should have the 'DroneCAN' bit enabled
 
-Note: Do not enable rc_override or rc_channels in the mLRS configuration when using RC Channels via DroneCAN as it adds unnecessary traffic on the bus.
+***Note***: Do not enable rc_override or rc_channels in the mLRS receiver when using DroneCAN as it adds substantial but unnecessary traffic on the CAN bus.
```

Comment:
Small tweak 'Do not enable rc_override or rc_channels in the mLRS receiver settings when using DroneCAN as it adds substantial and unnecessary traffic on the CAN bus.'

Reply:
done